tests/postgres_tests/test_bulk_update.py

```diff
@@ -44,3 +45,12 @@ def test_bulk_update(self):
                 self.assertSequenceEqual(
                     Model.objects.filter(**{field: new}), instances
                 )
+
+    def test_bulk_create(self):
+        """Minimal example of a field with a custom placeholder, but
+        a default dbtype."""
```

Comment:
I think you can drop this docstring as the code is pretty self-documenting. We don't use test comments for _what_ the test does but _why_ it does so only when necessary. The git commit and the link the PR and ticket is good enough context already.

```suggestion
```